pom.xml

```diff
@@ -373,18 +335,21 @@
       </plugin>
     </plugins>
   </build>
-
-  <reporting>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-javadoc-plugin</artifactId>
-        <version>3.4.1</version>
-        <configuration>
-          <show>public</show>
-          <source>8</source>
-        </configuration>
-      </plugin>
-    </plugins>
-  </reporting>
+  <profiles>
+    <profile>
+      <id>windows-ci</id>
+      <activation>
+        <os>
+          <family>Windows</family>
+        </os>
+        <property>
+          <name>env.CI</name>
+        </property>
+      </activation>
+      <properties>
+        <!-- TODO fails on CI on Windows for inscrutable reasons -->
+        <spotless.check.skip>true</spotless.check.skip>
+      </properties>
+    </profile>
+  </profiles>
```

Comment:
https://github.com/jenkins-infra/helpdesk/issues/3865

Reply:
This message does not state the reason why this PR was not approved, or the action that needed to be taken in order for this PR to be approved.

Reply:
commit ce65d9f